onnxruntime/src/tensor/ort_owned_tensor.rs

```diff
@@ -25,18 +23,18 @@ use crate::{
 #[derive(Debug)]
 pub struct OrtOwnedTensor<'t, 'm, T, D>
 where
-    T: TypeToTensorElementDataType + Debug + Clone,
+    T: TensorDataToType,
```

Comment:
`TensorDataToType` includes `Debug`, and `Clone` didn't seem to be needed any longer so I ditched it